src/pages/ProductDetail/Checkout/components/SwitchAuction.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const SwitchAuction = props => {
+  return (
+    <div>
```

Comment:
```suggestion
    <>
```
fragment로 처리해줘도 되는 부분인가요?